File: frontend/src/shared/enums/backend/command/run-command-preset-enum.ts
export const RunCommandPresetEnum = {
  CUSTOM: "CUSTOM",
  KILL_PROCESS: "KILL_PROCESS",
  IS_PROCESS_RUNNING: "IS_PROCESS_RUNNING",
  READ_CLIPBOARD: "READ_CLIPBOARD",
  WRITE_CLIPBOARD: "WRITE_CLIPBOARD",
  CHECK_INTERNET: "CHECK_INTERNET",
  SHUTDOWN_IN: "SHUTDOWN_IN",
  CANCEL_SHUTDOWN: "CANCEL_SHUTDOWN",
} as const;

export type RunCommandPresetEnum =
  (typeof RunCommandPresetEnum)[keyof typeof RunCommandPresetEnum];

File: backend/Core/Enums/Command/RunCommandPresetEnum.cs
namespace Core.Enums
{
    /// <summary>
    /// A ready made command. The step stores the preset and its one parameter rather than the
    /// rendered text, so improving a preset later fixes every flow already using it.
    /// </summary>
    public enum RunCommandPresetEnum
    {
        CUSTOM,
        KILL_PROCESS,
        IS_PROCESS_RUNNING,
        READ_CLIPBOARD,
        WRITE_CLIPBOARD,
        CHECK_INTERNET,
        SHUTDOWN_IN,
        CANCEL_SHUTDOWN,
    }
}
